applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_solver_wrapper.py

```diff
@@ -160,6 +160,8 @@ def _GetDataCommunicator(self):
     def _GetDefaultParameters(cls):
         return KM.Parameters("""{
             "type"                    : "",
+            "analysis_module"         : "",
+            "analysis_name"           : "",
```

Comment:
I wouldn't introduce this field. After assuming that each Python module implements only one class, you can "pascalize" the module name to take it (OFC we are assuming a naming convention such that `fsi_analysis` module implements `FsiAnalysis` class). Note that this is what we already did in order to unify the `MainKratos.py`.

Reply:
I do this because there are some which are not consistent in our side (few files, but anyway). I can remove it, for me is simpler, but this is more generic